### PR TITLE
Added common test: rviz_rendering

### PR DESCRIPTION
**Pull request by ahcorde**

### CODE REVIEW

File: rviz_rendering/include/rviz_rendering/objects/effort_visual.hpp [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
- * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
- * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
- * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
- * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
- * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
- * POSSIBILITY OF SUCH DAMAGE.
- */
+// Copyright (c) 2023, Open Source Robotics Foundation, Inc.
+// All rights reserved.
+// Redistribution and use in source and binary forms, with or without
```
Comment (clalancette) [inline rviz_rendering/include/rviz_rendering/objects/effort_visual.hpp:4]:
```suggestion

// Redistribution and use in source and binary forms, with or without
```

File: rviz_rendering/src/rviz_rendering/objects/effort_visual.cpp [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
- * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
- * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
- * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
- * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
- * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
- * POSSIBILITY OF SUCH DAMAGE.
- */
+// Copyright (c) 2023, Open Source Robotics Foundation, Inc.
+// All rights reserved.
+// Redistribution and use in source and binary forms, with or without
```
Comment (clalancette) [inline rviz_rendering/src/rviz_rendering/objects/effort_visual.cpp:4]:
```suggestion

// Redistribution and use in source and binary forms, with or without
```

File: rviz_rendering/src/rviz_rendering/objects/screw_visual.cpp [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
- * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
- * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
- * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
- * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
- * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
- * POSSIBILITY OF SUCH DAMAGE.
- */
+// Copyright (c) 2023, Open Source Robotics Foundation, Inc.
+// All rights reserved.
+// Redistribution and use in source and binary forms, with or without
```
Comment (clalancette) [inline rviz_rendering/src/rviz_rendering/objects/screw_visual.cpp:4]:
```suggestion

// Redistribution and use in source and binary forms, with or without
```

File: rviz_rendering/test/rviz_rendering/mesh_shape_test.cpp [hunk 4]
```diff
@@ -1,31 +1,31 @@
...
- * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
- * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
- * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
- * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
- * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
- * POSSIBILITY OF SUCH DAMAGE.
- */
+// Copyright (c) 2023, Open Source Robotics Foundation, Inc.
+// All rights reserved.
+// Redistribution and use in source and binary forms, with or without
```
Comment (clalancette) [inline rviz_rendering/test/rviz_rendering/mesh_shape_test.cpp:4]:
```suggestion

// Redistribution and use in source and binary forms, with or without
```

File: rviz_rendering/test/rviz_rendering/objects/effort_visual_test.cpp [hunk 5]
```diff
@@ -1,31 +1,31 @@
...
- * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
- * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
- * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
- * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
- * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
- * POSSIBILITY OF SUCH DAMAGE.
- */
+// Copyright (c) 2023, Open Source Robotics Foundation, Inc.
+// All rights reserved.
+// Redistribution and use in source and binary forms, with or without
```
Comment (clalancette) [inline rviz_rendering/test/rviz_rendering/objects/effort_visual_test.cpp:4]:
```suggestion

// Redistribution and use in source and binary forms, with or without
```

**Review by clalancette — CHANGES_REQUESTED** [review]
A few small nits to fix, then this looks good.

**Comment by ahcorde** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/21457/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/15743/)
* Linux-rhel [Build Status](http://ci.ros2.org/job/ci_linux-rhel/958/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/22177/)

**Review by clalancette — APPROVED** [review]

**Pull request merged**